style.css

```diff
@@ -24,83 +27,71 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
 	overflow:auto;
 	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
 	float:right;
 }
 
 
-
+/* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
 ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 970px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
 	z-index: 5;
 }
 
-ul li #a{
-	padding: 0;
-	position: absolute;
-	top: 58px;
-	left: 1100px;
-	width: 100px;
-	display: none;
-	visibility: hidden;
-	z-index: 5;
-}
 
 	
 ul li ul li{
-	background-color:  white;
+	background-color:  #0C59CF;
```

Comment:
I change the background color of the like in the drop down menu column to black because I has decided to make the nav bar background to be navy blue and the fact that these link belong to the nav bar so their background should be navy blue too.